src/uucore/src/lib/features/format/num_format.rs

```diff
@@ -293,13 +290,7 @@ impl Formatter for Float {
 
         let precision = match precision {
             Some(CanAsterisk::Fixed(x)) => x,
-            None => {
-                if matches!(variant, FloatVariant::Shortest) {
-                    6
-                } else {
-                    0
-                }
-            }
+            None => 6,
```

Comment:
could you please document what 6 means here ?

thanks

Reply:
Sure. That's part of the commits in #7384 though (so I'll update there and rebase).

Reply:
Done and rebased.